code/game/machinery/computer/camera.dm

```diff
@@ -10,75 +10,157 @@
 	var/mapping = 0 // For the overview file (overview.dm), not used on this page
 
 	var/list/network = list()
-	var/list/available_networks = list()
-	var/list/watchers = list() //who's using the console, associated with the camera they're on.
-
-/obj/machinery/computer/security/New() // Lists existing networks and their required access. Format: available_networks[<name>] = list(<access>)
-	generate_network_access()
-	..()
-
-/obj/machinery/computer/security/proc/generate_network_access()
-	available_networks["SS13"] =              list(ACCESS_HOS,ACCESS_CAPTAIN)
-	available_networks["Telecomms"] =         list(ACCESS_HOS,ACCESS_CAPTAIN)
-	available_networks["Research Outpost"] =  list(ACCESS_RD,ACCESS_HOS,ACCESS_CAPTAIN)
-	available_networks["Mining Outpost"] =    list(ACCESS_QM,ACCESS_HOP,ACCESS_HOS,ACCESS_CAPTAIN)
-	available_networks["Research"] =          list(ACCESS_RD,ACCESS_HOS,ACCESS_CAPTAIN)
-	available_networks["Prison"] =            list(ACCESS_HOS,ACCESS_CAPTAIN)
-	available_networks["Labor Camp"] =        list(ACCESS_HOS,ACCESS_CAPTAIN)
-	available_networks["Interrogation"] =     list(ACCESS_HOS,ACCESS_CAPTAIN)
-	available_networks["Atmosphere Alarms"] = list(ACCESS_CE,ACCESS_HOS,ACCESS_CAPTAIN)
-	available_networks["Fire Alarms"] =       list(ACCESS_CE,ACCESS_HOS,ACCESS_CAPTAIN)
-	available_networks["Power Alarms"] =      list(ACCESS_CE,ACCESS_HOS,ACCESS_CAPTAIN)
-	available_networks["Supermatter"] =       list(ACCESS_CE,ACCESS_HOS,ACCESS_CAPTAIN)
-	available_networks["MiniSat"] =           list(ACCESS_RD,ACCESS_HOS,ACCESS_CAPTAIN)
-	available_networks["Singularity"] =       list(ACCESS_CE,ACCESS_HOS,ACCESS_CAPTAIN)
-	available_networks["Anomaly Isolation"] = list(ACCESS_RD,ACCESS_HOS,ACCESS_CAPTAIN)
-	available_networks["Toxins"] =            list(ACCESS_RD,ACCESS_HOS,ACCESS_CAPTAIN)
-	available_networks["Telepad"] =           list(ACCESS_RD,ACCESS_HOS,ACCESS_CAPTAIN)
-	available_networks["TestChamber"] =       list(ACCESS_RD,ACCESS_HOS,ACCESS_CAPTAIN)
-	available_networks["ERT"] =               list(ACCESS_CENT_SPECOPS_COMMANDER,ACCESS_CENT_COMMANDER)
-	available_networks["CentComm"] =          list(ACCESS_CENT_SECURITY,ACCESS_CENT_COMMANDER)
-	available_networks["Thunderdome"] =       list(ACCESS_CENT_THUNDER,ACCESS_CENT_COMMANDER)
```

Comment:
All of this has been removed, and networks are now decided by the boards of the subtypes, since having a selection box for networks in this UI size would not work well, and we have a search anyway.